crates/sui-graphql-rpc/src/types/epoch.rs

```diff
@@ -12,52 +12,121 @@ use super::transaction_block::{TransactionBlock, TransactionBlockFilter};
 use super::validator_set::ValidatorSet;
 use async_graphql::connection::Connection;
 use async_graphql::*;
+use sui_indexer::models_v2::epoch::StoredEpochInfo;
+use sui_types::sui_system_state::sui_system_state_summary::SuiValidatorSummary;
 
-#[derive(Clone, Debug, PartialEq, Eq, SimpleObject)]
-#[graphql(complex)]
+// #[derive(Clone, Debug, PartialEq, Eq, SimpleObject)]
+// #[graphql(complex)]
+#[derive(Clone, Debug)]
 pub(crate) struct Epoch {
-    /// The epoch's id as a sequence number that starts at 0 and it is incremented by one at every epoch change
-    pub epoch_id: u64,
+    pub stored_epoch_info: StoredEpochInfo,
+    pub validators_summary: Vec<SuiValidatorSummary>,
```

Comment:
do we need this as an additional field on Epoch? Since it's only used for resolving validator_set, wonder if we could drop it off Epoch and just resolve it there

Reply:
removed!